src/Interfaces/MarketingSuite/Campaign.php

```diff
@@ -40,6 +40,7 @@
  * @property bool $send_sms
  * @property Carbon $print_request
  * @property Carbon $sms_at
+ * @property bool $send_postcard
```

Comment:
It's so dumb that the campaign interface has all the retention properties typehinted. MAKES ZERO SENSE.